### ISSUE TITLE
RuboCop warning Style/EmptyElse for :css only

### ISSUE THREAD

**Issue by michsch** [causes/haml-lint#68]
If i use conditions containing only CSS, i got the following warning:

```bash
[W] RuboCop: Style/EmptyElse: Redundant empty `else`-clause.
```

```haml
- if Rails.env == 'development'
  :css
    .jw-non-functional {
      box-shadow: 0 0 6px 4px rgba(255, 127, 80, .6);
    }
- else
  :css
    .jw-non-functional {
      display: none;
    }
```

I've created a gist: https://gist.github.com/michsch/c876a01e576fb7e1de63

Expected result: no warning
Current result: [W] RuboCop: Style/EmptyElse: Redundant empty \`else\`-clause.

Our work around at the moment in ```haml-lint.yml```:
```yml
  RuboCop:
    ignored_cops:
      - Style/EmptyElse
```

**Comment by sds** [comment]
Thanks for the report, @michsch!

Fixed in ee07324cef85b45ce9c9dd2cea33882c9841ab6f.
